torchtune/modules/transformer.py

```diff
@@ -129,7 +129,7 @@ def __init__(
         num_heads: int,
         head_dim: int,
         norm: nn.Module,
-        output: nn.Linear,
+        output: nn.Module,
```

Comment:
Can we change this back to nn.Linear? I don't think we need the general nn.Module in your latest version (lmk if I'm missing something here though)

Reply:
@SalmanMohammadi sorry one more thing on this.. can you also change the corresponding docstring back to nn.Linear too? Then I promise we are good to merge 😄

Reply:
my bad! I had a thought that the pre-commits sometimes catch things like this?

Reply:
Yeah I'm also confused why it wasn't caught tbh..